### PR TITLE
Hello-1 from dev1_change2

### PR DESCRIPTION
**Pull request by isac-fernandesdev1**
New change - Hello-1 logic from dev1_change2 branch

### CODE REVIEW

File: HelloABC.html [hunk 1]
```diff
@@ -33,5 +33,6 @@
 Hi-2 from dev1_change1
 Hi-3 from dev1_change1
 Hi-5 from master
+Hello-1 from dev1_change2
```
Comment (isac-fernandesdev1) [inline HelloABC.html]: Please add Hello-2 logic

Comment (isac-fernandesdev1) [reply HelloABC.html]: Added Hello-2 logic.

**Review by isac-fernandesdev1 — COMMENTED** [review]
Hello-2 logic looks good.

File: HelloABC.html [hunk 2]
```diff
@@ -33,5 +33,7 @@
...
+Hello-2 from dev1_change2
```
Comment (isac-fernandes) [inline HelloABC.html]: Please add Hello-3 logic after this.

**Review by isac-fernandes — CHANGES_REQUESTED** [review]
Hello-3 logic is mandatory

File: HelloABC.html (hunk 2 above)
Comment (isac-fernandesdev1) [reply HelloABC.html]: Hello-3 logic added, thanks for the catch.

**Review by isac-fernandesdev1 — COMMENTED** [review]
Hello-3 added as requested

**Pull request merged**